internal/persistence/persistors/mysqlstore/store_organization_test.go

```diff
@@ -0,0 +1,496 @@
+package mysqlstore
+
+import (
+	"context"
+	"github.com/dembygenesis/local.tools/internal/model"
+	"github.com/dembygenesis/local.tools/internal/model/modelhelpers"
+	"github.com/dembygenesis/local.tools/internal/persistence/database_helpers/mysql/assets/mysqlmodel"
+	"github.com/dembygenesis/local.tools/internal/persistence/database_helpers/mysql/mysqlhelper"
+	"github.com/dembygenesis/local.tools/internal/persistence/database_helpers/mysql/mysqltx"
+	"github.com/dembygenesis/local.tools/internal/persistence/persistors/mysqlstore/testhelper"
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/volatiletech/null/v8"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+	"strings"
+	"testing"
+	"time"
+)
+
+type testCaseGetOrganizations struct {
+	name       string
+	filter     *model.OrganizationFilters
+	mutations  func(t *testing.T, db *sqlx.DB)
+	assertions func(t *testing.T, db *sqlx.DB, paginated *model.PaginatedOrganizations, err error)
+}
+
+func getTestCasesGetOrganizations() []testCaseGetOrganizations {
+	return []testCaseGetOrganizations{
+		{
+			name: "success-filter-ids-in",
+			filter: &model.OrganizationFilters{
+				IdsIn: []int{1},
+			},
+			mutations: func(t *testing.T, db *sqlx.DB) {
+				entry := mysqlmodel.Organization{
+					ID:            1,
+					Name:          "TEST",
+					CreatedBy:     null.IntFrom(2),
+					LastUpdatedBy: null.IntFrom(3),
+					CreatedAt:     time.Now(),
+					LastUpdatedAt: null.TimeFrom(time.Now()),
+					IsActive:      true,
+				}
+				err := entry.Insert(context.Background(), db, boil.Infer())
+				require.NoError(t, err, "error inserting sample data")
+			},
+			assertions: func(t *testing.T, db *sqlx.DB, paginated *model.PaginatedOrganizations, err error) {
+				require.NoError(t, err, "unexpected error")
+				require.NotNil(t, paginated, "unexpected nil paginated")
+				require.NotNil(t, paginated.Organizations, "unexpected nil organizations")
+				require.NotNil(t, paginated.Pagination, "unexpected nil pagination")
+				assert.True(t, len(paginated.Organizations) == 1, "unexpected number of organizations")
+				assert.True(t, paginated.Pagination.RowCount == 1, "unexpected row count")
+
+				modelhelpers.AssertNonEmptyOrganizations(t, paginated.Organizations)
+			},
+		},
+		{
+			name: "success-filter-names-in",
+			filter: &model.OrganizationFilters{
+				OrganizationNameIn: []string{"Organization A", "Organization B"},
+			},
+			mutations: func(t *testing.T, db *sqlx.DB) {
+				entry := []mysqlmodel.Organization{
+					{ID: 1,
```

Comment:
dont put properties inline like **{ID: 1**

Reply:
Alright. This is now resolved.